docs/application-connector/assets/metadataapi.yaml

```diff
@@ -376,6 +388,7 @@ components:
         url:
           type: 'string'
           format: 'uri'
+          description: 'URL to fetch OAuth access token'
```

Comment:
```suggestion
          description: 'URL to fetch an OAuth access token'
```

Reply:
Since articles etc. are usually omitted in code, and this file is a duplication of an already merged API spec description, let's leave it as it is for consistency's sake:)